dbscripts/xml/upgrade.xml

```diff
@@ -115,6 +119,11 @@
 		<!-- Only new 3.4 installations need this fix -->
 		<migration class="PKP\migration\upgrade\v3_4_0\I8592_SiteNotificationSubscriptions"/>
 		<migration class="APP\migration\upgrade\v3_4_0\I9822_ChangeUsageStatsTemporaryTablesIndexes"/>
+		<migration class="PKP\migration\upgrade\v3_5_0\I10962_UpdateEmailTemplateVariables"/>
```

Comment:
If I understand it correctly , here upgrading from min `3.1.0.0` to max `3.4.0.1` and upgrading from 

1. `3.3.0-x` to `3.5.0-x` 
2. `3.4.0-x` to `3.5.0-x` 
3. `3.5.0-0/3.5.0-1` to `3.5.0-x` 

will not have this block of migration to run . so do we need it here ? please correct me if I am mistaken .